hnsecure/src/test/java/ca/bc/gov/hlth/hnsecure/test/TestMessages.java

```diff
@@ -0,0 +1,39 @@
+package ca.bc.gov.hlth.hnsecure.test;
+
+/**
+ * We use sample messages in JUnit testing.
+ * This class contains all the test messages.
+ *  New messages can be added for more tests
+ * @author pankaj.kathuria
+ *
+ */
+
+public class TestMessages {
```

Comment:
An alternative to using messages stored as Strings (with their associated formatting challenges) would be to have sample messages stored as files and loaded in the test case. Probably not necessary at this point but it we ever build up a sizable catalog of test messages then files will be much simpler to maintain.

Reply:
agree.. it will be much easier to maintain if stored as files and reusable, if we want to use similar messages in ReadyAPI